lib/gui/app.js

```diff
@@ -198,6 +198,11 @@ app.config(($provide) => {
   });
 });
 
+app.config(($qProvider) => {
+  // Prevent errors generated by $uibModalInstance.dismiss() unhandled rejections
+  $qProvider.errorOnUnhandledRejections(false);
```

Comment:
I think it is only related to the modals. I'll take another look tomorrow and handle it wherever it's necessary. For now I know that when closing by using the 'X' (https://github.com/resin-io/etcher/blob/master/lib/gui/components/tooltip-modal/controllers/tooltip-modal.js#L36) is an issue.